### ISSUE TITLE
patch boot.img failed

### ISSUE THREAD

**Issue by fettdrac** [topjohnwu/Magisk#1042]
my device is Samsung Galaxy Tab A 9.7(SM-P550) P550ZCU1BQI1 armv7
here is my boot.img zipped
[boot_bqi1.zip](https://github.com/topjohnwu/Magisk/files/2830752/boot_bqi1.zip)
when I was trying to patch the image by Manager app it outputed this log:
[magisk_install_log_20190205_112824.log](https://github.com/topjohnwu/Magisk/files/2830765/magisk_install_log_20190205_112824.log)
I have given the runtime permission of read&write extrnal storage.
There is a strange condition that the app can patch the image successfully on a X86 android simlutor!
Hope your early reply!

**Comment by fettdrac** [comment]
If the image that patched on X86 can work well on my device ,please close the issue!

**Comment by domotic-29** [comment]
Same problem on SM-G361F.
I started the process to install Magisk few days ago with versions V6. I had created the patched boot file that I had to install on the phone.
I installed the new of Magisk Manager (V7). There is now a loop where the system keeps telling me there is a new version of Magisk Manager available (although V7 is installed).
Anyway, I tried to regenerate the patched boot file with V7, and it failed.
